### PR TITLE
fix(listbox): footer is now external to the list

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-720

Little oopsie on my part when I MVPied the ListBox: the footer need to always be visible at the bottom of the list but it was not 🙈 

Before:

https://user-images.githubusercontent.com/63734941/170720236-bda71312-6417-43b6-8044-e96fd8c5bfd4.mov

divider:
image

After:
image

Divider style: 
image

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by gdostie — APPROVED** [review]

**Comment by aiheon** [comment]
I pushed another fix for the divider in this branch I realized the issue was coming from Plasma and not the Admin 😅

**Review by GermainBergeron — APPROVED** [review]

**Review by toofff — APPROVED** [review]

File: packages/react/src/components/itemBox/ItemBox.tsx [hunk 1]
```diff
@@ -87,11 +87,13 @@ export class ItemBox extends Component<IItemBoxProps> {
                     role="option"
                     onMouseDown={(event) => event.preventDefault()}
                 >
-                    {this.props.prepend ? <Content {...this.props.prepend} /> : null}
-                    <PartialStringMatch partialMatch={this.props.highlight} caseInsensitive>
-                        {this.props.displayValue || this.props.value}
-                    </PartialStringMatch>
-                    {this.props.append ? <Content {...this.props.append} /> : null}
+                    <span className="truncante-item-box-content">
```
Comment (gdostie) [inline packages/react/src/components/itemBox/ItemBox.tsx:90]:
```suggestion
                    <span className="truncate-item-box-content">
```

File: packages/style/scss/components/list-box.scss [hunk 2]
```diff
@@ -30,18 +32,22 @@
...
     color: var(--color);
-    white-space: nowrap;
-    text-overflow: ellipsis;
     list-style: none;
     background-color: var(--background-color);
     outline: none;
-
     cursor: var(--cursor);
 
+    > .truncante-item-box-content {
```
Comment (gdostie) [inline packages/style/scss/components/list-box.scss:45]:
```suggestion
    > .truncate-item-box-content {
```

Comment (gdostie) [reply packages/style/scss/components/list-box.scss:45]: Also why not use the existing `truncate` class?

https://github.com/coveo/plasma/blob/master/packages/style/scss/typography/utility.scss#L66-L72

Comment (aiheon) [reply packages/style/scss/components/list-box.scss:45]: Oh! I should go in the Utility more often 🙈

**Pull request merged**